core/forms.py

```diff
@@ -17,4 +17,4 @@ class SuggestionForm(forms.ModelForm):
     """
     class Meta:
         model = Suggestion
-        fields = ('breast_symmetry', 'breast_tissue', 'breast_placement', 'bra_wire')
+        fields = ('breast_symmetry', 'breast_tissue', 'breast_placement', 'bra_padding', 'bra_wire',)
```

Comment:
Are we going to take out breast symmetry? 
And rename breast_tissue to shape?

Reply:
I changed breast_tissue to breast_shape. Do we need the symmetry stuff for the bra suggestion ajax?